src/data/script/ConstraintChangerModPlugin.java

```diff
@@ -79,12 +155,69 @@ public class ConstraintChangerModPlugin extends BaseModPlugin {
+        LunaToRealKeymap.put(FIELD_AUTOMATED_SHIPS_OP_THRESHOLD, "RKZ_skill_technology_AutomatedShips_OPThreshold");
+        // Industry
+        LunaToRealKeymap.put(FIELD_BULK_TRANSPORT_CARGO_CAPACITY_MAX_PERCENTAGE, "RKZ_skill_industry_BulkTransport_CargoCapacityMaxPercent");
+        LunaToRealKeymap.put(FIELD_BULK_TRANSPORT_CARGO_CAPACITY_THRESHOLD, "RKZ_skill_industry_BulkTransport_CargoCapacityThreshold");
+        LunaToRealKeymap.put(FIELD_BULK_TRANSPORT_FUEL_CAPACITY_MAX_PERCENTAGE, "RKZ_skill_industry_BulkTransport_FuelCapacityMaxPercent");
+        LunaToRealKeymap.put(FIELD_BULK_TRANSPORT_FUEL_CAPACITY_THRESHOLD, "RKZ_skill_industry_BulkTransport_FuelCapacityThreshold");
+        LunaToRealKeymap.put(FIELD_BULK_TRANSPORT_PERSONNEL_CAPACITY_MAX_PERCENTAGE, "RKZ_skill_industry_BulkTransport_PersonnelCapacityMaxPercent");
+        LunaToRealKeymap.put(FIELD_BULK_TRANSPORT_PERSONNEL_CAPACITY_THRESHOLD, "RKZ_skill_industry_BulkTransport_PersonnelCapacityThreshold");
+        LunaToRealKeymap.put(FIELD_BULK_TRANSPORT_BURN_BONUS, "RKZ_skill_industry_BulkTransport_BurnBonus");
+
+        LunaToRealKeymap.put(FIELD_FIELD_REPAIRS_MIN_HULL, "RKZ_skill_industry_FieldRepairs_MinHull");
+        LunaToRealKeymap.put(FIELD_FIELD_REPAIRS_MAX_HULL, "RKZ_skill_industry_FieldRepairs_MaxHull");
+        LunaToRealKeymap.put(FIELD_FIELD_REPAIRS_MIN_CR, "RKZ_skill_industry_FieldRepairs_MinCR");
+        LunaToRealKeymap.put(FIELD_FIELD_REPAIRS_MAX_CR, "RKZ_skill_industry_FieldRepairs_MaxCR");
+        LunaToRealKeymap.put(FIELD_FIELD_REPAIRS_REPAIR_RATE_BONUS, "RKZ_skill_industry_FieldRepairs_RepairRateBonus");
+        LunaToRealKeymap.put(FIELD_FIELD_REPAIRS_INSTA_REPAIR_PERCENT, "RKZ_skill_industry_FieldRepairs_InstaRepairPercent");
+
+        LunaToRealKeymap.put(FIELD_CONTAINMENT_PROCEDURES_CREW_LOSS_REDUCTION, "RKZ_skill_industry_ContainmentProcedures_CrewLossReduction");
+        LunaToRealKeymap.put(FIELD_CONTAINMENT_PROCEDURES_FUEL_SALVAGE_BONUS, "RKZ_skill_industry_ContainmentProcedures_FuelSalvageBonus");
+        LunaToRealKeymap.put(FIELD_CONTAINMENT_PROCEDURES_FUEL_USE_REDUCTION_MAX_PERCENT, "RKZ_skill_industry_ContainmentProcedures_FuelUseReductionMaxPercent");
+        LunaToRealKeymap.put(FIELD_CONTAINMENT_PROCEDURES_FUEL_USE_REDUCTION_MAX_FUEL, "RKZ_skill_industry_ContainmentProcedures_FuelUseReductionMaxFuel");
+
+        LunaToRealKeymap.put(FIELD_DERELICT_OPERATIONS_DP_COST_REDUCTION_PER_DMOD, "RKZ_skill_industry_DerelictOperations_DPCostReductionPerDmod");
+
+        // Shared
+        LunaToRealKeymap.put(FIELD_SHARED_FIELD_REPAIRS_AND_CONTAINMENT_PROCEDURES_OP_THRESHOLD, "RKZ_skill_industry_FieldRepairs_and_ContainmentProcedures_OPThreshold");
+        LunaToRealKeymap.put(FIELD_SHARED_CREW_TRAINING_AND_FLUX_REGULATION_OP_THRESHOLD, "RKZ_skill_leadership_CrewTraining_and_FluxRegulation_OPThreshold");
+        LunaToRealKeymap.put(FIELD_SHARED_FIGHTER_BAYS, "RKZ_skill_leadership_FighterUplink_and_CarrierGroup_FighterBays");
```

Comment:
bloated, but unfortunatelly one of the pains we'll have to live with. moving the bloat somewhere else (e.g. some method ran in a `static { ... }` block) won't help reduce it.